src/test/java/ua/lviv/iot/graph/GraphTest.java

```diff
@@ -0,0 +1,60 @@
+package ua.lviv.iot.graph;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class GraphTest {
+	private Graph<String> graph;
+
+	public GraphTest() {
+		graph = new Graph<>();
+	}
+
+	public void testVertexAdding() {
+		graph.addVertex("visa");
+		graph.addVertex("foreignpassport");
+		graph.addVertex("visa");
+		graph.addVertex("hotel");
+		graph.addVertex("visa");
+		graph.addVertex("bankstatement");
+		graph.addVertex("bankstatement");
+		graph.addVertex("nationalpassport");
+		graph.addVertex("hotel");
+		graph.addVertex("creditcard");
+		graph.addVertex("creditcard");
+		graph.addVertex("nationalpassport");
+		graph.addVertex("nationalpassport");
+		graph.addVertex("birthcertificate");
+		graph.addVertex("foreignpassport");
+		graph.addVertex("nationalpassport");
+		graph.addVertex("foreignpassport");
+		graph.addVertex("militarycertificate");
+		graph.addVertex("militarycertificate");
+		graph.addVertex("nationalpassport");
+		assertEquals(graph.numberOfVertices(), 8);
+	}
+
+	@Test
+	public void testEdgeAdding() {
+		testVertexAdding();
```

Comment:
Good idea, but not very good implementation. You would never want you unit tests depend on each other. If you do similar things in 2 test – you would need a third helper method that's not marked as @Test